src/packages/client/src/generation/TSClient.ts

```diff
@@ -333,12 +328,13 @@ path.join(__dirname, 'schema.prisma');
 // https://github.com/microsoft/TypeScript/issues/3192#issuecomment-261720275
 function makeEnum(x) { return x; }
 
-${new Enum({
-  name: 'ModelName',
-  values: this.dmmf.mappings.modelOperations.map((m) => m.model)
-}).toJS()}
+${this.dmmf.schema.enumTypes.prisma.map((type) => new Enum(type, true).toJS()).join('\n\n')}
+${this.dmmf.schema.enumTypes.model?.map((type) => new Enum(type, false).toJS()).join('\n\n') ?? ''}
```

Comment:
I don't understand yet the need to have these 2, one .prisma with arg true, one with model with arg false 🤔

Reply:
It's to generate the `exports.Prisma = ` namespace prefix or not